src/test/java/org/opentripplanner/netex/mapping/ServiceLinkMapperTest.java

```diff
@@ -83,34 +100,77 @@ public void mapServiceLinks() {
     quayIdByStopPointRef.add("RUT:StopPoint:2", "NSR:Quay:2");
     quayIdByStopPointRef.add("RUT:StopPoint:3", "NSR:Quay:3");
 
-    ServiceLinkMapper serviceLinkMapper = new ServiceLinkMapper(
-      new FeedScopedIdFactory("RB"),
-      new DataImportIssueStore(false)
+    EntityById<Stop> stopsById = new EntityById<>();
+
+    DataImportIssueStore issueStore = new DataImportIssueStore(false);
+    StopMapper stopMapper = new StopMapper(ID_FACTORY, issueStore);
+    StopPattern.StopPatternBuilder stopPatternBuilder = StopPattern.create(3);
+
+    Station parentStation = Station
+      .of(ID_FACTORY.createId("NSR:StopArea:1"))
+      .withName(NonLocalizedString.ofNullable("Parent Station"))
```

Comment:
Do you also prefer those static builder methods instead of `new`? If so we could create `NonLocalizedString.of`.